### DISCUSSION TITLE
how to enable GPU support in KubeEdge? how to use k8s-device-plugins in edge?

### DISCUSSION THREAD

**Discussion by crush-wu** [kubeedge/kubeedge#4907] [Q&A]
I created a k8s-device-plugins pod in edge node, but i can't read the nvidia.com/gpu information.

**Comment by chenqiangzhishen** [comment]
have the same requirement

**Comment by J1a-wei** [comment]
+1

**Comment by Shelley-BaoYue** [comment]
rootdirectory default value is `/var/lib/edged` but not `/var/lib/kubelet`, you try using `/var/lib/edged/device-plugins` in pod config or modify rootdirectory to `/var/lib/kubelet` in edgecore.yaml
